### ISSUE TITLE
Documentation or example

### ISSUE THREAD

**Issue by JeanLoriston** [jkoelker/python-tradeking#2]
do you have some docs to explain how to use some of the functions or can you give some example code how to place a options order

**Comment by JohnForeman** [comment]
yes, i would like to see some examples also.  I'm trying to use VS Code and i cant get it to import tradeking.  what am i doing wrong
